yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -611,65 +612,66 @@ def pixelize_cylinder(np.float64_t[:,:] buff,
         rbounds[0] = 0.0
     r_inc = 0.5 * fmin(dx, dy)
 
-    for i in range(radius.shape[0]):
-        r0 = radius[i]
-        theta0 = theta[i]
-        dr_i = dradius[i]
-        dtheta_i = dtheta[i]
-        # Skip out early if we're offsides, for zoomed in plots
-        if r0 + dr_i < rbounds[0] or r0 - dr_i > rbounds[1]:
-            continue
-        theta_i = theta0 - dtheta_i
-        theta_inc = r_inc / (r0 + dr_i)
-
-        while theta_i < theta0 + dtheta_i:
-            r_i = r0 - dr_i
-            costheta = math.cos(theta_i)
-            sintheta = math.sin(theta_i)
-            while r_i < r0 + dr_i:
-                if rmax <= r_i:
+    with nogil:
```

Comment:
Is this a case where we'd want to use
```python
for i in prange(radius.shape[0], nogil=True):
   pass
```

Reply:
sure! I think prange should work here. but i'll wait until the rest is actually working....

Reply:
ok well using `prange` might be a bit more work than I want to put in right now -- lots of compilation errors along the lines of :

```python-traceback
      yt/utilities/lib/pixelization_routines.pyx:635:20: Cannot read reduction variable in loop body
      
      Error compiling Cython file:
      ------------------------------------------------------------
      ...
                  while r_i < r0 + dr_i:
                      if rmax <= r_i:
                          r_i += r_inc
                          continue
                      y = r_i * costheta
                      x = r_i * sintheta
                          ^
      ------------------------------------------------------------
      
      yt/utilities/lib/pixelization_routines.pyx:636:20: Cannot read reduction variable in loop body
```